awesomemotive/affiliatewp-affiliate-info
# Issue/34 - Bump WordPress minimum requirements

Issue #34


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Affiliate Info add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Create a new page with the shortcode `[affiliate_info_name]`, visit the page with an affiliate link and confirm it shows the affiliate name
